README.md

```diff
@@ -156,6 +156,113 @@ Use `getNextPage` to paginate through the observations.
  Use `queryDeviceObservations(deviceId, query, callback)` for querying device observations.
 ```
 
+### Get Device Templates
+This will return the list of device templates.
+
+```js
+  starfish.getDeviceTemplates((err, data) => {
+    if(err) {
+      console.log("Error:", err)
+    } else {
+      console.log("Return devices: ", data)
+    }
+  })
+```
+
+Response format in case of success:
+
+```js
+  {
+    deviceTemplates: [
+      {
+        id: 'someid',
+        name: 'templateName',
+        sensors: ['few', 'sensors']
+      }
+    ]
+  }
+```
+
+### Get count of Device Templates by `count`
```

Comment:
If `count` is part of `getDeviceTemplates` response, then we can include an query string `{count:true}` or else this becomes a new API.